recipes/darwin-toolchain/all/conanfile.py

```diff
@@ -0,0 +1,119 @@
+from conans import ConanFile, tools
+
+import os
+import platform
+import copy
+
+
+class DarwinToolchainConan(ConanFile):
+    name = "darwin-toolchain"
+    version = "1.0.9"
+    license = "Apple"
+    settings = "os", "arch", "build_type", "os_build"
+    options = {"bitcode": [True, False]}
+    default_options = {"bitcode": True}
+    description = "Darwin toolchain to (cross) compile for macOS/iOS/watchOS/tvOS"
+    homepage = "https://developer.apple.com"
+    topics = ("xcode", "bitcode", "darwin")
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = "darwin-toolchain.cmake"
+
+    @property
+    def cmake_system_name(self):
+        if self.settings.os == "Macos":
+            return "Darwin"
+        return str(self.settings.os)
+
+    @property
+    def cmake_system_processor(self):
+        return {"x86": "i386",
+                "x86_64": "x86_64",
+                "armv7": "arm",
+                "armv8": "aarch64"}.get(str(self.settings.arch))
+
+    def config_options(self):
+        # build_type is only useful for bitcode
+        if self.settings.os == "Macos":
+            del self.settings.build_type
+            del self.options.bitcode
+
+    def validate(self):
+        # We export recipes on a Linux machine, thus we have to rely on os_build and not sys.platform
+        if self.settings.os_build != "Macos":
+            raise ConanInvalidConfiguration("Build machine must be Macos")
+        if not tools.is_apple_os(self.settings.os):
+            raise ConanInvalidConfiguration("os must be an Apple os")
```

Comment:
```suggestion
        if not tools.is_apple_os(self.settings.os):
            raise ConanInvalidConfiguration("os must be an Apple os")
        if tools.cross_building(self):
            raise ConanInvalidConfiguration("Build machine must be Macos")
```

Reply:
So many leftovers from our in-house stuff, sorry 😬